.travis.yml

```diff
@@ -0,0 +1,21 @@
+---
+language: python
+python:
+  - "3.5"
```

Comment:
Any particular requirement for the extra python build?

Reply:
Wasn't able to get the Windows build to be recognized without it

Reply:
Still? Oh, I thought that was resolved.

Reply:
That worked fine, thanks! I guess having the python version on its own is what caused the issue previously.